src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildManager.kt

```diff
@@ -206,10 +206,10 @@ object CargoBuildManager {
 
     fun isBuildConfiguration(configuration: CargoCommandConfiguration): Boolean {
         val args = ParametersListUtil.parse(configuration.command)
-        return when (val command = args.firstOrNull()) {
+        return when (val command = args.firstOrNull { !it.startsWith("+") }) {
```

Comment:
As I can see, we use similar code several times
Maybe it's worth introducing some helper method to parse cargo command and return something like
```kotlin
data class ParsedCommand(
    val command: String,
    val toolchain: String?,
    val additionalArguments: List<String>
)
```
I suppose it should greatly simplify code reading since it splits parsing logic from "run configuration" logic